site/gatsby-site/src/components/checklists/CheckListForm.js

```diff
@@ -109,6 +162,10 @@ export default function CheckListForm({
     );
   };
 
+  const addRisk = (newRisk) => {
+    setFieldValue('risks', [newRisk].concat(values.risks || []));
```

Comment:
I'm wary of all the `|| []`. Nothing to do here right now but we are hiding empty values all over the place. Please consider this worth coming back to, after this re-factor.

Reply:
I had previously added a bunch of `|| []`s because we were getting crashes in unexpected, hard-to-diagnose circumstances. I would like to be able to guarantee that we'll never get a null where we're not expecting one, but I don't really know how to that in JavaScript. This seems like something there would be best practices about, but I don't really know what they are beyond the fact that things like this are considered suspect.